src/content/ccip/architecture.mdx

```diff
@@ -155,7 +159,11 @@ Execution delays in excess of one hour should be rare as a result of _Smart Exec
 
 ## CCIP rate limits
 
-Chainlink CCIP token transfers benefit from rate limits for additional security. A rate limit has a maximum capacity and a refill rate, which is the speed at which the maximum capacity is restored after a token transfer has consumed some or all of the available capacity. The rate limits are enforced at both the source and destination blockchains for maximum security. If these rate limits are reached, descriptive errors with detailed information are generated and returned to the sender. This enables CCIP users to gracefully handle these errors within their dApps to preserve the end-user experience. A comprehensive list of errors and their descriptions is available on the [errors API reference](/ccip/api-reference/errors#ratelimiter) page.
+Chainlink CCIP token transfers benefit from rate limits for additional security. A rate limit has a maximum capacity and a refill rate, which is the speed at which the maximum capacity is restored after a token transfer has consumed some or all of the available capacity.
```

Comment:
Let's pick a specific lane to use as an example here

Reply:
I added a link to the specific example lane and edited the text to more accurately describe the example.